server/views/pages/contacts/manage/contactDetails/details.njk

```diff
@@ -14,7 +22,7 @@
 
     <div class="govuk-grid-column-two-thirds">
         <span class="govuk-caption-l">Manage Contacts</span>
-        <h1 class="govuk-heading-l">Contact details - {{ contact.firstName | capitaliseName + " " + contact.lastName | capitaliseName }}</h1>
+        <h1 class="govuk-heading-l">Contact details - {{ contact.firstName | capitaliseName + " " + contact.middleNames | capitaliseName + " " + contact.lastName | capitaliseName }}</h1>
```

Comment:
Did you just add a nunjucks filter for this? Should probably change it over but will approve anyway and let you decide.

Reply:
I have just wrapped the middleNames in a if condition in case it is null to avoid having an extra space

Reply:
I see what you mean the  `njkEnv.addFilter('formatName', formatName) `to be used like in the `prisonerSearchResults` `{{ result | formatName }}`

Let me add that with new format name method

Reply:
` {{ contact | reverseFormatName }}` filter added